### PR TITLE
Removed dev env declarations

### PR DESCRIPTION
**Pull request by farisdurrani**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,14 +5,14 @@
...
+    "installf": "cd frontend/playground-frontend && npm install",
     "installb": "conda --version && cd conda && conda env create -f environment.yml & conda env update -f environment.yml",
-    "startf": "cd frontend/playground-frontend && dotenv set MODE=dev && npm start",
+    "startf": "cd frontend/playground-frontend && npm start",
     "startb": "conda activate dlplayground && python -m backend.driver",
     "secrets": "conda activate dlplayground && python -m backend.aws_helpers.aws_secrets_utils.build_env",
     "secrets:deploy": "python -m backend.aws_helpers.aws_secrets_utils.build_env",
     "docker:build": "docker build -t dlp .",
-    "prod:variable": "cd frontend/playground-frontend && dotenv set MODE prod"
+    "prod:variable": "npm install -g dotenv-cli && cd frontend/playground-frontend && dotenv set MODE prod"
```
Comment (karkir0003) [inline package.json:15]: do we know if this for sure works?

Comment (karkir0003) [reply package.json:15]: @farisdurrani

**Comment by karkir0003** [comment]
also `git pull origin dev` @farisdurrani

**Review by karkir0003 — APPROVED** [review]

**Comment by farisdurrani** [comment]
No changes

**Pull request closed without merging**